addons/binding/org.openhab.binding.windcentrale/ESH-INF/thing/millThing.xml

```diff
@@ -57,49 +58,55 @@
 		<item-type>Number</item-type>
 		<label>Wind speed</label>
```

Comment:
Does the also deserve a dimension or is there none

Reply:
We can use the quantity javax.measure.quantity.Speed for it. But I don't now the unit Beaufort is available.

Reply:
Currently there is no Beaufort unit in SmartHomeUnits. Beaufort is also a scale and not a unit. 
Should Beaufort also be added to the SmartHomeUnits @htreu, @kaikreuzer?
If so, what would be the dimension? Speed, WindSpeed (new) or Dimensionless?

Reply:
The mathematical formula seems to be a sort of one-way function from a speed in km/h or kn to Bft. When having 5Bft one can not ensure a correct conversion to a real speed.
That being said I don't see a valid conversion from Bft into any other unit, so I suggest to not add it to the SmartHomeUnits.